plugins/keycloak-backend/README.md

```diff
@@ -54,33 +61,43 @@ catalog:
         clientSecret: ${KEYCLOAK_CLIENTSECRET}
 ```
 
+Optionally, you can override the default parameters by adding the basic configuration to `app-config.yaml`.
+
+```yaml
+keycloak:
+  userQuerySize: 500
+  groupQuerySize: 250
+```
```

Comment:
I don't like that we're introducing another place where the keycloak is configured. Why can't the `userQuerySize` and `groupQuerySize` live together with the rest of the config in `catalog.providers.keycloakOrg.<env>`